reporter/gen_report.js

```diff
@@ -32,11 +35,22 @@ Object.keys(originals).forEach(key => {
       escape(atob(originals[key]))));
 });
 
+var fileWriter = new FileWriter(false);
+
+fileWriter.writeFile = function writeFile(file, callback) {
+  if (file.length > 255) {
```

Comment:
Well the way I see we have 2 cases:

1. The case were the full path doesn't take more than 255 (which don't need slice)
1. The ones that do need slice...

Then in `the ones that do need slice`, my logic was to remove the excess from the path's basename without the extension. But I believe it can be more performant if we get the extension and do a slice - the length of the extension (and then join it with the extension).

Will revisit it.